website/static/css/markdown.css

```diff
@@ -0,0 +1,74 @@
+.post .postHeader .postHeaderTitle {
+  color: #151c3d;
+  font-size: 35px;
+  font-weight: normal;
+}
+
+.post article h1,
+.post article h2,
+.post article h3,
+.post article h4,
+.post article h5 {
+  color: #151c3d;
+  font-weight: normal;
+}
+
+.post article h2::after {
+  content: '';
+  display: block;
+  margin-top: 15px;
+  margin-bottom: 5px;
+  width: 76px;
+  height: 3px;
+  border-radius: 1.5px;
+  background: #4bcee5;
+  left: 0;
+}
+
+.post article h2.after0::after {
+  background: #4bcee5;
+}
+.post article h2.after1::after {
+  background: #24dbb4;
+}
+.post article h2.after2::after {
+  background: #f95d31;
+}
+.post article h2.after3::after {
+  background: #7f55cc;
+}
+.post article h2.after4::after {
+  background: #fea430;
+}
+.post article h2.after5::after {
+  background: #0774d3;
+}
```

Comment:
can this be achieved with `nth-child`? if so, we can probably get rid of the `dynamic-styles.js` stuff :P

Reply:
can you make that loop? the 7th child should be color 0, 8th color 1 etc. if it can loop i would rather get rid of the .js

Reply:
`:nth-child(6n+0)`

Reply:
OR we make a rule that you only get 6 H2 elements. no exceptions :P

Reply:
^ I think

Reply:
will try that.